.github/workflows/ci.yml

```diff
@@ -0,0 +1,82 @@
+name: Invidious CI
+
+on: [push, pull_request]
```

Comment:
This is just my opinion, but, I would normally have this be pushes to master only and all pull requests. With this running on all pushes and pull requests I believe it runs twice.

Reply:
How can this be fixed?

Reply:
This instead
```yml
on:
  push:
    branches: "master"
  pull_request:
    branches: "*"
```

Reply:
Great, thanks!